### PR TITLE
Use travis-ci conditional builds to create feedstocks in linux

### PR DESCRIPTION
**Pull request by isuruf**
Follow up for https://github.com/conda-forge/staged-recipes/pull/4176

It seems the boolean language needs the parenthesis every where and "IS" is not supported yet.

cc @jakirkham

### CODE REVIEW

**Comment by isuruf** [comment]
This finally worked. Forks and PRs run on OSX and creating feedstocks runs in Linux.

**Comment by jakirkham** [comment]
Nice work @isuruf! Thanks for pursuing this. Will look over it more tomorrow, but am glad to hear that you found a working configuration. 🙂

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,15 @@
...
-osx_image: xcode6.4
+jobs:
+  include:
+    - if: (fork IN (true)) OR (type IN (pull_request))
+      os: osx
+      osx_image: xcode6.4
+
+    - if: (fork IN (false)) AND (NOT (type IN (pull_request)))
+      os: linux
+      sudo: false
```
Comment (jakirkham) [inline .travis.yml]: We should be able to have `sudo: false` regardless, right? Admittedly I don't think it matters much for macOS, but still.

Comment (isuruf) [reply .travis.yml]: `sudo: false` is there to select which type of linux container. Not used for`OSX at all

Comment (jakirkham) [reply .travis.yml]: Right. Though we do have `sudo: false` a few lines below already. Maybe we should drop one of these lines.

**Comment by mwcraig** [comment]
Thanks for this @isuruf  and @jakirkham!

**Pull request merged**